scripts/imgtool/image.py

```diff
@@ -0,0 +1,164 @@
+"""
```

Comment:
Yeah, it allows you to control what happens when someone imports *. If you ever add more code to this init file, those definitions would end up in the namespace of anybody that imports * on this package. Maybe that's not a problem though.